src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting.Azure/JsonSerialization/AzureStorageJson.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
+using static Microsoft.Extensions.AI.Evaluation.Reporting.Storage.AzureStorageResponseCache;
+
+namespace Microsoft.Extensions.AI.Evaluation.Reporting.JsonSerialization;
+
+internal static partial class AzureStorageJson
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1716:Identifiers should not match keywords", Justification = "Default matches the generated source naming convention.")]
+    internal static class Default
+    {
+        private static JsonSerializerOptions? _options;
+        internal static JsonSerializerOptions Options => _options ??= CreateJsonSerializerOptions(writeIndented: true);
+        internal static JsonTypeInfo<CacheEntry> CacheEntry => Options.GetTypeInfo<CacheEntry>();
+        internal static JsonTypeInfo<ScenarioRunResult> ScenarioRunResult => Options.GetTypeInfo<ScenarioRunResult>();
```

Comment:
Can we rename these to `CacheEntryTypeInfo` and `ScenarioRunResultTypeInfo`. That would read slightly better at call sites which currently just say `AzureJson.Default.CacheEntry`.